cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -32,6 +32,7 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 
 	flags.Uint64("update-parallelism", 0, "task update parallelism (0 = all at once)")
 	flags.String("update-delay", "0s", "delay between task updates (0s = none)")
+	flags.String("update-on-failure", "pause", "action on failure during update (pause|continue)")
```

Comment:
Shouldn't this be for update only since it's part of the UpdateConfig?
Also name is a bit ambiguous... `on-update-failure` seems more precise, though still not great.